components/Banner/examples/tests/snapshot/MDCBannerSnapshotTests.m

```diff
@@ -36,9 +37,13 @@ - (void)setUp {
 
   // Uncomment below to recreate all the goldens (or add the following line to the specific
   // test you wish to recreate the golden for).
-  //  self.recordMode = YES;
+//    self.recordMode = YES;
 
-  self.bannerView = [[MDCBannerView alloc] initWithFrame:CGRectMake(0, 0, 350, 51)];
+  self.bannerView = [[MDCBannerView alloc] initWithFrame:CGRectZero];
+  UIEdgeInsets margins = UIEdgeInsetsZero;
```

Comment:
These look like more than "fixing" the tests - can you please fix the PR description?

Reply:
Sure. Updated the description.

Reply:
The description is still remarking on only API changes and golden updates. This appears to be code changes.

Reply:
Updated with "uses layoutMargins to set the banner view's margin" to make it clear. The code changes here were caused by the API support changes too.